src/components/MovieListPage/movieListPage.jsx

```diff
@@ -1,19 +1,20 @@
 import React, { useState, useEffect } from 'react';
+import { useSearchParams, Outlet, useNavigate } from "react-router-dom";
 import SearchForm from '../Search/searchForm';
 import Dialog from '../Dialog/dialog';
 import MovieForm from '../MovieForm/movieForm';
 import GenreSelect from '../Genre/genreSelect';
 import { genreList } from '../Genre/genreList';
 import SortControl from '../SortControl/sortControl';
 import MovieTile from '../MovieTile/movieTile';
-import MovieDetails from '../MovieDetails/movieDetails';
 import '../MovieForm/movie-form.css';
 import './movie-list-page.css';
 
 const MovieListPage = (props) => {
     const [showAddDialog, setShowAddDialog] = useState(false);
     const [sortCriterion, setSortCriterion] = useState('release_date'); // sort criterion
     const [selectedMovie, setSelectedMovie] = useState({}); // selected movie
```

Comment:
redundant, not using selectedMovie value anywhere

Reply:
Done.